src/keys.rs

```diff
@@ -326,6 +333,36 @@ impl From<FullViewingKey> for SpendValidatingKey {
 }
 
 impl FullViewingKey {
+    /// Creates a `FullViewingKey` from a `SpendingKey` and `SpendValidatingKey`.
+    /// This is necessary for FROST key management.
+    ///
+    /// Note: See [FROST Book - Technical details](https://frost.zfnd.org/zcash/technical-details.html)
+    #[cfg(feature = "unstable-frost")]
+    pub fn from_sk_and_ask(sk: &SpendingKey, ask: SpendValidatingKey) -> FullViewingKey {
+        FullViewingKey {
+            ak: ask,
```

Comment:
`ask` is the `SpendAuthorizingKey` which is the private key that matches the `ak` (`SpendValidatingKey`) so it's probably better to follow the spec nomenclature

```suggestion
    #[cfg(feature = "unstable-frost")]
    pub fn from_sk_and_ak(sk: &SpendingKey, ak: SpendValidatingKey) -> FullViewingKey {
        FullViewingKey {
            ak,
```

Reply:
oh I always confuse the two. Not really good at acronyms. Thanks!